### PR TITLE
Quick fixes: Clean up test code

### PR DESCRIPTION
**Pull request by dougbu**
- remove useless `configuration` variables and `Configuration` instances
- remove "Review" code comment (the only product code change)
- reduce repeated code in `TestHelper` for functional tests

### CODE REVIEW

**Comment by dougbu** [comment]
FYI only; will push changes once builds settle

File: test/WebSites/ActionConstraintsWebSite/Startup.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
-            var configuration = app.GetTestConfiguration();
+            app.GetTestConfiguration();
```
Comment (rynowak) [inline test/WebSites/ActionConstraintsWebSite/Startup.cs]: So what does this do now? presumably magic? Can we name it accordingly `app.AddTestServices()` or something reasonable?

Comment (dougbu) [inline test/WebSites/ActionConstraintsWebSite/Startup.cs]: renamed it to `UseCultureReplacer()` since it adds just one middleware

**Comment by dougbu** [comment]
@rynowak updated

**Comment by rynowak** [comment]
@dougbu solid :shipit: once we get the all-clear.

**Comment by dougbu** [comment]
fc1017f

**Pull request closed without merging**